<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>616</width>
    <height>466</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>616</width>
    <height>466</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Coverletter Creator</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="mainTabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabPersonal">
       <attribute name="title">
        <string>Personal Info</string>
       </attribute>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>First Name</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="le_firstName"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Last Name</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Address</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="le_lastName"/>
        </item>
        <item row="2" column="1">
         <widget class="QTextEdit" name="te_personalAddress"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Mobile</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Email</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Homepage</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Github</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Linkedin</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="le_mobile"/>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="le_email"/>
        </item>
        <item row="5" column="1">
         <widget class="QLineEdit" name="le_homepage"/>
        </item>
        <item row="6" column="1">
         <widget class="QLineEdit" name="le_githubName"/>
        </item>
        <item row="7" column="1">
         <widget class="QLineEdit" name="le_LinkedinName"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabCompany">
       <attribute name="title">
        <string>Company Info</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QFormLayout" name="formLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Company Name</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="le_companyName">
            <property name="toolTip">
             <string>Full company name, for example: Intel Deutschland GmbH</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Company Short Name</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="le_companyShortName">
            <property name="toolTip">
             <string>Company Short name, for example: Intel</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Department</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="le_department">
            <property name="text">
             <string>Human Resources</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>Address</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QTextEdit" name="te_companyAddress"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QFormLayout" name="formLayout_4">
          <item row="0" column="0">
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>Letter Title</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="le_Lettertitle">
            <property name="text">
             <string>Job Application for</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>Job Title</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_15">
            <property name="text">
             <string>Job Referrence ID</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="le_jobTitle"/>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="le_jobRefId"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Line" name="line_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label_16">
            <property name="text">
             <string>Recipient</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="receipientGender">
            <item>
             <property name="text">
              <string>Mr.</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Ms.</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Unknown</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLineEdit" name="le_RecepientName"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabAboutMe">
       <attribute name="title">
        <string>About me</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item alignment="Qt::AlignHCenter|Qt::AlignTop">
         <widget class="QLabel" name="label_17">
          <property name="text">
           <string>Give a short introduction about yourself</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="te_aboutMe"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabWhyFirm">
       <attribute name="title">
        <string>Why this Firm?</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item alignment="Qt::AlignHCenter|Qt::AlignTop">
         <widget class="QLabel" name="label_18">
          <property name="text">
           <string>Why did you choose this particular firm?</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="te_WhyFirm"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabWhyYou">
       <attribute name="title">
        <string>Why You?</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item alignment="Qt::AlignHCenter|Qt::AlignTop">
         <widget class="QLabel" name="label_19">
          <property name="text">
           <string>Specify why you are the perfect fit for this job?</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="te_whyYou"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabFinish">
       <attribute name="title">
        <string>Finish</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="1" column="0">
           <widget class="QLabel" name="label_21">
            <property name="text">
             <string>Enclosing</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_20">
            <property name="text">
             <string>Letter Closing</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="le_enclosing">
            <property name="text">
             <string>Attached </string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" colspan="2">
           <widget class="QLineEdit" name="le_closing">
            <property name="text">
             <string>Sincerely,</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <widget class="QCheckBox" name="cb_CV">
              <property name="text">
               <string>Curriculum Vitae</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="cb_Transcripts">
              <property name="text">
               <string>Transcripts</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="cb_certificates">
              <property name="text">
               <string>Certificates</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="cb_referenceLetters">
              <property name="text">
               <string>Reference Letters</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pb_generatePdf">
            <property name="text">
             <string>Generate PDF</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pb_generateText">
            <property name="text">
             <string>Generate Text</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>616</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionReset_All"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionReset_All">
   <property name="text">
    <string>Reset All</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
